docs/assets/css/layout.less

```diff
@@ -0,0 +1,7 @@
+// taken from enhancements/layout.less
```

Comment:
Could add " in cfgov-refresh" as was the comment in the global header less file.

Reply:
Good call.